template/lambda-sqs-worker-cdk/infra/appStack.ts

```diff
@@ -51,32 +54,129 @@ export class AppStack extends Stack {
       encryptionMasterKey: kmsKey,
     });
 
+    topic.addSubscription(new aws_sns_subscriptions.SqsSubscription(queue));
```

Comment:
Interesting, this stack deploys the _source_ topic. That feels a bit strange as typically you'd subscribe to an external source 😅

Reply:
Yeah I've been playing with this today. Looks like we can create a subscription to a topic which seems more ideal.

<img width="417" alt="image" src="https://github.com/seek-oss/skuba/assets/18017094/40ba32a9-3e45-4706-ace2-f6b43183f3cf">